charts/argocd-apps/dev-values.yaml

```diff
@@ -52,3 +52,6 @@ workflows:
 
 graphProxy:
   enabled: true
+
+dashboard:
+  enabled: false
```

Comment:
Do we not want this enabled?

Reply:
I thought it might not work anyway to look at the dashboard if the namespace is not `workflows` that's why I disabled it, but happy to enable

Reply:
Okay, will enable it then